src/Sarif.Viewer.VisualStudio/ErrorListService.cs

```diff
@@ -223,6 +225,33 @@ private void WriteRunToErrorList(Run runLog)
             }
         }
 
+        private IEnumerable<IEnumerable<AnnotatedCodeLocation>> CreateAnnotationsFromStacks(IList<Stack> stacks)
+        {
+            List<List<AnnotatedCodeLocation>> codeLocationCollections = new List<List<AnnotatedCodeLocation>>();
+
+            foreach (Stack stack in stacks)
+            {
+                if (stack.Frames == null) { continue; }
+
+                List<AnnotatedCodeLocation> codeLocations = new List<AnnotatedCodeLocation>();
+
+                foreach (StackFrame stackFrame in stack.Frames)
+                {
+                    codeLocations.Add(new AnnotatedCodeLocation()
```

Comment:
Nit: the parens on a default ctor call aren't needed when you use property initializer syntax.